test/statistics/StatisticsTest.js

```diff
@@ -190,36 +190,41 @@ describe('Statistics with negative period', function () {
             var cfg = new Config.ClientConfig();
             cfg.properties['hazelcast.client.statistics.enabled'] = true;
             cfg.properties['hazelcast.client.statistics.period.seconds'] = -2;
-            return Client.newHazelcastClient(cfg);
+            return Client.newHazelcastClient(cfg)
         }).then(function (cl) {
             client = cl;
         });
     });
 
     after(function () {
         client.shutdown();
-        RC.shutdownCluster(cluster.id);
+        return RC.shutdownCluster(cluster.id);
     });
 
     it('should be enabled via configuration', function () {
         return Util.promiseWaitMilliseconds(1000).then(function () {
-            return getClientStatisticsFromServer(cluster);
+            return getClientStatisticsFromServer(cluster, client);
         }).then(function (stats) {
             expect(stats).to.not.equal('');
         });
     });
 });
 
-function getClientStatisticsFromServer(cluster) {
-    var deferred = DeferredPromise();
-    var script = 'client0=instance_0.getClientService().getConnectedClients().' +
-        'toArray()[0]\nresult=client0.getClientStatistics();';
-    RC.executeOnController(cluster.id, script, 1).then(function (response) {
+function getClientStatisticsFromServer(cluster, client) {
+    var clientUuid = client.getClusterService().uuid;
+    console.log(clientUuid);
```

Comment:
Do you want to keep this here?

Reply:
leftover, good catch

Reply:
This PR needs to be merged assap because all builds constantly fail. I'll merge it this way and remove the leftover in another PR.